src/engine/engine.cpp

```diff
@@ -416,7 +416,7 @@ const char* DivEngine::getSystemName(DivSystem sys) {
       return "Neo Geo Extended Channel 2";
     // Furnace-specific systems
     case DIV_SYSTEM_AY8910:
-      return "AY-3-8910";
+      return "Atari ST/MSX";
```

Comment:
There are too many systems using the AY-3-8910 chip. By this naming scheme, it would be more like:
"ZX Spectrum/Pentagon/Atari ST/MSX/Intellivision/Amstrad CPC/Vectrex/Old Arcade/Slot Machine/Electronika BK"...

Reply:
idk, revert it to AY-3-8910?

Reply:
Yep, could be an option.